src/tablet/tablet_impl.cc

```diff
@@ -3507,6 +3507,7 @@ void TabletImpl::GetTableFollower(RpcController* controller, const ::openmldb::a
     if (info_map.empty()) {
         response->set_msg("has no follower");
         response->set_code(::openmldb::base::ReturnCode::kNoFollower);
+        return;
```

Comment:
what's the bug behavior like ?

Reply:
no follower but return ok, rewrite msg and code by code below